conf/messages.en

```diff
@@ -132,7 +132,7 @@ vat.direct.debit.setup=Set up a Direct Debit (opens in a new window or tab)
 
 partial.heading = VAT
 partial.vrn = Your VAT registration number (VRN) is {0}.
-partial.moredetails = More VAT details
+partial.more_details = More VAT details
```

Comment:
Why add underscore? Seems to be inconsistent. We use camel case in other places. Is this simply a 'partial refactor'?

Reply:
We use `partial.more_details` on corporation-tax-frontend. Changed it to `more_details` here for consistency.